pkg/minikube/download/iso.go

```diff
@@ -142,3 +143,85 @@ func downloadISO(isoURL string, skipChecksum bool) error {
 
 	return download(urlWithChecksum, dst)
 }
+
+func WindowsISO(windowsIsoURL string) error {
+
+	u, err := url.Parse(windowsIsoURL)
+	if err != nil {
+		return errors.Wrapf(err, "url.parse %q", windowsIsoURL)
+	}
+
+	// It's already downloaded
+	if u.Scheme == fileScheme {
+		return nil
+	}
+
+	// Lock before we check for existence to avoid thundering herd issues
+	dst := filepath.Join(detect.ISOCacheDir(), "Windows_Server_2022_EVAL_64")
```

Comment:
Instead of hardcoding the iso file name you can use something like this to get the filename which will make this reusable and easier to integrate into the existing iso download function as opposed to duplicating the functionality for windows only

```go
windowsISOUrl := "https://go.microsoft.com/fwlink/p/?LinkID=2195280&clcid=0x409&culture=en-us&country=US"
r, _ := http.Head(windowsISOUrl)
_, params, err := mime.ParseMediaType(r.Header.Get("Content-Disposition"))
fmt.Println(params["filename"], err)
```

Reply:
Thanks, added this logic in the change!